plugins/modules/mount.py

```diff
@@ -899,11 +899,6 @@ def main():
                     module.fail_json(
                         msg="Error unmounting %s: %s" % (name, msg))
 
-            if os.path.exists(name):
```

Comment:
This will change the behavior of ```state: absent```.
But I don't think the ```rmdir``` logic should be skipped. Because it breaks backward compatibility.
If you want to skip it, how about adding ```keep_mountpoint``` or ```preserve_mountpoint``` to keep the mountpoint to avoid ```Directory not empty``` error instead of removing this section.

If this is set as ```true```, you can skip ```rmdir``` section. Also, if the default value is set as ```false```, to keep backward compatibility, you can set it to ```false``` by default.